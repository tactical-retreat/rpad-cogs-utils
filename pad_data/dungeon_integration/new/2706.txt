Wallace Descended! - Seventh Dragon Caller-Annihilation

stage 0

3539 - Flame Wing Wisedragon, Nerva @ level 10
Fire damage halved
Reduce damage from Fire by 50%
Fireform
Reduce damage from all sources by 75% for 3 turns (1 time use)
----- Turn 1 -----
Burn Claw
32370 Damage - Movetime 50% for 1 turns, Deal 300% damage (1 time use)
----- Turn 1 -----
Vasahal Vorzen
539500 Damage - Deal 5000% damage (10 hits, 500% each)
----- Turn 2 -----
Flash Burn
32370 Damage - Blind all orbs on the board, Deal 300% damage (1 time use)
----- Turn 2 -----
Vasahal Vorzen
539500 Damage - Deal 5000% damage (10 hits, 500% each)
----- Turn 3 -----
Margo Cult + Crimson Breath
Voids status ailments for 999 turns + Enemy action (1 time use)
----- Turn 3 -----
Margo Cult + Lava Flare
Voids status ailments for 999 turns + Change all orbs to Fire (1 time use)
----- Turn 3 -----
Vasahal Vorzen
539500 Damage - Deal 5000% damage (10 hits, 500% each)
----- Turn 4 -----
Fireform + Giant Iceflame Talons
Reduce damage from all sources by 75% for 3 turns + Bind 1 random for 4 turns (1 time use)
----- Turn 4 -----
Vasahal Vorzen
539500 Damage - Deal 5000% damage (10 hits, 500% each)
----- Turn 5 -----
Pyro Breath
38844 Damage - Deal 360% damage (3 hits, 120% each) (1 time use)
----- Turn 5 -----
Vasahal Vorzen
539500 Damage - Deal 5000% damage (10 hits, 500% each)
----- Turn 6 -----
Freeze Ignis
32370 Damage - Lock 15 random Fire, Water, Wood, Light, Dark, Heal, Jammer, Poison, Mortal Poison orbs, Deal 300% damage (1 time use)
----- Turn 6 -----
Vasahal Vorzen
539500 Damage - Deal 5000% damage (10 hits, 500% each)
----- Turn 7 -----
Crimson Breath
377650 Damage - Deal 3500% damage (5 hits, 700% each) (1 time use)
----- Turn 7 -----
Lava Flare
37765 Damage - Change all orbs to Fire, Deal 350% damage (1 time use)
----- Turn 7 -----
Vasahal Vorzen
539500 Damage - Deal 5000% damage (10 hits, 500% each)
----- Turn 8 -----
Fireform + Giant Iceflame Talons
Reduce damage from all sources by 75% for 3 turns + Bind 1 random for 4 turns (1 time use)
----- Turn 8 -----
Vasahal Vorzen
539500 Damage - Deal 5000% damage (10 hits, 500% each)
----- Turn 9 -----
Inferno Breath
48555 Damage - Deal 450% damage (3 hits, 150% each) (1 time use)
----- Turn 9 -----
Vasahal Vorzen
539500 Damage - Deal 5000% damage (10 hits, 500% each)
----- Turn 10 -----
Inferno Breath
48555 Damage - Deal 450% damage (3 hits, 150% each) (1 time use)
----- Turn 10 -----
Vasahal Vorzen
539500 Damage - Deal 5000% damage (10 hits, 500% each)
----- HP <= 100 -----
Vasahal Vorzen
539500 Damage - Deal 5000% damage (10 hits, 500% each)

stage 1

3260 - Yamato Flame Dragon Caller, Tsubaki @ level 10
Scarlet Sky Robes + Dragonbind Kotodama + Flame Camellia
Voids status ailments for 999 turns + Bind Dragon cards for 10 turns + Enemy action
----- HP <= 100 -----
Fire Guardian's Sunblade
34374 Damage - Reduce damage from all sources by 50% for 1 turns, Deal 320% damage (50% chance)
Firechain Thrust
34374 Damage - Bind active skills for 5 turns, Deal 320% damage
----- HP <= 100 -----
Firechain Thrust
34374 Damage - Bind active skills for 5 turns, Deal 320% damage
----- HP <= 50 -----
Vanishing Lifespring - Red Sun + Crimson Blossom Storm
Change all orbs to Jammer + Absorb combo <= 6 damage for 5 turns (1 time use)
----- HP <= 49 -----
Banishing Firebrand
85936 Damage - Change Jammer to Fire, Deal 800% damage
----- HP <= 20 -----
Soaring Camellia Flame
290034 Damage - Deal 2700% damage (9 hits, 300% each)

stage 2

3576 - Seventh Dragon Caller, Shapeshifter Wallace @ level 10
Wood damage halved
Reduce damage from Wood by 50%
Concealed Mountain Path + Guardian Dragon Formation + Mucus Skin
Void damage >= 10000000 for 999 turns + Voids status ailments for 999 turns + Reduce damage from all sources by 75% for 7 turns (1 time use)
----- Turn 1 -----
Rainbow Cloak + Hell Poison Claw
Change own attribute to Fire + Change Random 7 to Poison (1 time use)
----- Turn 2 -----
Rainbow Cloak + Hell Venom Claw
Change own attribute to Water + Change Random 5 to Poison (1 time use)
----- Turn 3 -----
Rainbow Cloak + Paralysis Vine
Change own attribute to Wood + Bind 1 subs for 7 turns (1 time use)
----- Turn 4 -----
Rainbow Cloak + Poison Rave
Change own attribute to Light + Change Random 10 to Poison (1 time use)
----- Turn 5 -----
Rainbow Cloak + Hell Poison Rave
Change own attribute to Dark + Change Random 7 to Poison (1 time use)
----- Turn 6 -----
Hell Bloom
641400 Damage - Change Poison to Jammer, Deal 6000% damage (1 time use)
----- Turn 7 -----
The Allure of Swift Death + The Art of Contamination + Storm Rave
Mortal Poison skyfall +20% for 7 turns + Unable to Poison, Mortal Poison orbs for 2 turns + Enemy action (1 time use)
----- Turn 8 -----
Rainbow Cloak + Paralysis Vine
Change own attribute to random one of Fire, Water, Wood, Light, Dark + Bind 1 subs for 7 turns (1 time use)
----- Turn 9 -----
Thorned Husk Fang + Cyclone Rave
Locked Poison, Mortal Poison skyfall +100% for 5 turns + Enemy action (1 time use)
----- HP <= 100 -----
Rainbow Cloak + Seven Gardens
Change own attribute to random one of Fire, Water, Wood, Light, Dark + Change all orbs to Fire, Water, Wood, Light, Dark, Heal, Poison (45% chance)
Rainbow Cloak + Cyclone Rave
Change own attribute to random one of Fire, Water, Wood, Light, Dark + Enemy action (40% chance)
Rainbow Cloak + Dead Poison Rave
Change own attribute to random one of Fire, Water, Wood, Light, Dark + Change Random 14 to Poison (15% chance)
----- HP <= 15 -----
Rainbow Cloak + Everstorm Rainbow Blade
Change own attribute to random one of Fire, Water, Wood, Light, Dark + Enemy action

