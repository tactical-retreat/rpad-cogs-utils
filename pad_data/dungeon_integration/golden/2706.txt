Wallace Descended! - Seventh Dragon Caller-Annihilation

stage 0

3539 - Flame Wing Wisedragon, Nerva @ level 10
Fire damage halved
Reduce damage from Fire by 50%
Fireform
Reduce damage from all sources by 75% for 3 turns
----- HP <= 100 -----
Vasahal Vorzen
539500 Damage - Deal 5000% damage (10 hits, 500% each)

stage 1

3260 - Yamato Flame Dragon Caller, Tsubaki @ level 10
Scarlet Sky Robes + Dragonbind Kotodama + Flame Camellia
Voids status ailments for 999 turns + Bind Dragon cards for 10 turns + Enemy action
----- HP <= 100 -----
Fire Guardian's Sunblade
34374 Damage - Reduce damage from all sources by 50% for 1 turns, Deal 320% damage
Firechain Thrust
34374 Damage - Bind active skills for 5 turns, Deal 320% damage
----- HP <= 49 -----
Banishing Firebrand
85936 Damage - Change Jammer to Fire, Deal 800% damage
----- HP <= 50 -----
Vanishing Lifespring - Red Sun + Crimson Blossom Storm
Change all orbs to Jammer + Absorb combo <= 6 damage for 5 turns
----- HP <= 20 -----
Soaring Camellia Flame
290034 Damage - Deal 2700% damage (9 hits, 300% each)

stage 2

3576 - Seventh Dragon Caller, Shapeshifter Wallace @ level 10
Wood damage halved
Reduce damage from Wood by 50%
Concealed Mountain Path + Guardian Dragon Formation + Mucus Skin
Void damage >= 10000000 for 999 turns + Voids status ailments for 999 turns + Reduce damage from all sources by 75% for 7 turns
----- HP <= 100 -----
Rainbow Cloak + Seven Gardens
Change own attribute to random one of Fire, Water, Wood, Light, Dark + Change all orbs to Fire, Water, Wood, Light, Dark, Heal, Poison
Rainbow Cloak + Cyclone Rave
Change own attribute to random one of Fire, Water, Wood, Light, Dark + Enemy action
Rainbow Cloak + Dead Poison Rave
Change own attribute to random one of Fire, Water, Wood, Light, Dark + Change Random 14 to Poison
----- HP <= 15 -----
Rainbow Cloak + Everstorm Rainbow Blade
Change own attribute to random one of Fire, Water, Wood, Light, Dark + Enemy action

